RUFAS/data_structures/animal_to_manure_connection.py

```diff
@@ -0,0 +1,217 @@
+from dataclasses import dataclass
+from enum import Enum
+from RUFAS.enums import AnimalCombination
+
+
+class ManureStreamType(Enum):
+    """
+    Enumeration of the types of manure streams.
+
+    Attributes
+    ----------
+    PARLOR: int
+        Represents manure from the parlor.
+    GENERAL: int
+        Represents all manure other than what is deposited in or traveling to/from the milking parlor.
+    """
+
+    PARLOR = 0
+    GENERAL = 1
```

Comment:
```suggestion
    PARLOR : str
        Represents manure from the parlor.
    GENERAL : str
        Represents all manure other than what is deposited in or traveling to/from the milking parlor.

    """

    PARLOR = "parlor"
    GENERAL = "general"
```
IMO a string would be more appropriate here, because even though it is a little redundant there is not connection between parlor and 0 or general and 1

Reply:
Thank you for catching this Ed, it is an important specification!